src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java

```diff
@@ -17,7 +17,7 @@
 
     private static final String ACCOUNT_SID = "AC123";
     private static final String SIGNING_KEY_SID = "SK123";
-    private static final String SECRET = "secret";
+    private static final String SECRET = "secretsecretsecretsecretsecret00";
```

Comment:
I updated these to meet the minimum key length, and avoid [`WeakKeyException`s](https://github.com/jwtk/jjwt/blob/a7ee38053d34e8cd3a61430ec41405b13e640e54/api/src/main/java/io/jsonwebtoken/SignatureAlgorithm.java#L363). Do they merit an inline comment?

Reply:
I think it's fine w/o a comment.